src/silx/gui/plot/ImageView.py

```diff
@@ -338,6 +339,83 @@ def _actionTriggered(self, checked=False):
             self.plot.setSideHistogramDisplayed(checked)
 
 
+class AggregationModeAction(qt.QWidgetAction):
+    """Action providing few filters to the image"""
+
+    sigAggregationModeChanged = qt.Signal()
+
+    def __init__(self, parent):
+        qt.QWidgetAction.__init__(self, parent)
+
+        toolButton = qt.QToolButton(parent)
+
+        filterAction = qt.QAction(self)
+        filterAction.setText("No filter")
+        filterAction.setCheckable(True)
+        filterAction.setChecked(True)
+        filterAction.setProperty("aggregation", None)
```

Comment:
What about using the same type every where?
```suggestion
        filterAction.setProperty("aggregation", items.ImageDataAggregated.Aggregation.NONE)
```